src/mono/wasm/runtime/hybrid-globalization/common.ts

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+import { mono_wasm_new_external_root } from "../roots";
+import {MonoString } from "../types";
+import { Int32Ptr } from "../types/emscripten";
+import { js_string_to_mono_string_root } from "../strings";
+
+export function pass_exception_details(ex: any, exceptionMessage: Int32Ptr){
```

Comment:
Please use existing `wrap_error_root`. Note that the first argument could be null.